backend_addon/{{ cookiecutter.__folder_name }}/setup.py

```diff
@@ -59,15 +59,15 @@
         "plone.restapi",
         "plone.volto",
         {%- endif %}
+        {% if cookiecutter.__feature_headless == "0" and cookiecutter.plone_version >= "6.1" -%}
```

Comment:
A backend_addon does not necessarily need `plone.classicui`.

Reply:
True, but I think having the default demo content is expected.